libs/libjemalloc/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jemalloc
+PKG_VERSION:=5.3.0
+PKG_RELEASE:=1
+MAJOR_VERSION:=2
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/$(PKG_NAME)/$(PKG_NAME)/releases/download/$(PKG_VERSION)
+PKG_HASH:=2db82d1e7119df3e71b7640219b6dfe84789bc0537983c3b7ac4f7189aecfeaa
+
+PKG_LICENSE:=Public-Domain
+PKG_LICENSE_FILES:=COPYING
+
+PKG_MAINTAINER:=Philip Prindeville <philipp@redfish-solutions.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libjemalloc
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Jemalloc general purpose allocator with fragmentation avoidance and scalable concurrency
```

Comment:
I think this title is too long and because of that, it is not shown in make menuconfig.